spec/features/admin_views_attendence_spec.rb

```diff
@@ -42,6 +45,36 @@
     end
   end
 
+  scenario "admin sees total number of active students" do
+    admin = FactoryGirl.create(:user, role: "admin")
+    FactoryGirl.create(:user_with_answered_quiz)
+    FactoryGirl.create(:user_with_answered_quiz)
+
+    sign_in(admin)
+
+    click_link 'View Attendance'
+
+    within(".columns.medium-6:first-child") do
```

Comment:
this is probably a brittle selector - can we name the container with a class or ID to better isolate it and prevent this test from breaking if there are additional markup changes?